[0m13:00:23.063756 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F312747CB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F310789D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F313B3BD90>]}


============================== 13:00:23.074225 | 128d2c0f-6948-440a-aefd-61456e92dd6d ==============================
[0m13:00:23.074225 [info ] [MainThread]: Running with dbt=1.10.9
[0m13:00:23.075050 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True', 'use_colors': 'True', 'cache_selected_only': 'False', 'target_path': 'None', 'version_check': 'True', 'invocation_command': 'dbt init', 'log_format': 'default', 'warn_error': 'None', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'use_experimental_parser': 'False', 'quiet': 'False', 'log_cache_events': 'False', 'empty': 'None', 'indirect_selection': 'eager', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\pc\\.dbt', 'static_parser': 'True', 'write_json': 'True'}
[0m13:00:52.558475 [debug] [MainThread]: Starter project path: C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\include\starter_project
[0m13:00:52.659270 [info ] [MainThread]: 
Your new dbt project "ecommerce_site" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:00:52.661439 [info ] [MainThread]: Setting up your profile.
[0m13:06:19.712831 [error] [MainThread]: Encountered an error:

[0m13:06:19.884299 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\cli\requires.py", line 178, in wrapper
    result, success = func(*args, **kwargs)
                      ~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\cli\requires.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\cli\main.py", line 483, in init
    results = task.run()
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\task\init.py", line 346, in run
    self.setup_profile(profile_name)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\task\init.py", line 262, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\task\init.py", line 179, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\task\init.py", line 163, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\task\init.py", line 129, in generate_target_from_input
    target[key] = click.prompt(
                  ~~~~~~~~~~~~^
        text, default=default, hide_input=hide_input, type=type
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\click\termui.py", line 168, in prompt
    value = prompt_func(prompt)
  File "C:\Users\pc\AppData\Local\Programs\Python\Python313\Lib\site-packages\click\termui.py", line 151, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m13:06:19.888537 [debug] [MainThread]: Command `dbt init` failed at 13:06:19.888191 after 357.05 seconds
[0m13:06:19.889800 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F313BE2D70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F313D11250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F313D0C270>]}
[0m13:06:19.890410 [debug] [MainThread]: Flushing usage events
[0m13:06:21.617701 [debug] [MainThread]: An error was encountered while trying to flush usage events
